RubeesEat/Pages/Shared/_Layout.cshtml

```diff
@@ -4,12 +4,40 @@
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Rubees Eat</title>
-    <link rel="stylesheet" href="~/css/site.css"/>
+    <link rel="stylesheet" href="/css/site.css"/>
+    <link rel="stylesheet" href="/css/UserHomePage.css"/>
+    <link rel="stylesheet" href="/css/BillDetails.css"/>
     <link rel="icon" type="image/x-icon" href="~/favicon.ico">
 </head>
 <body>
-@RenderBody()
+<nav>
+    <button class="hamburger" id="hamburger">
+        <span></span>
+        <span></span>
+        <span></span>
+    </button>
+    <ul class="nav-menu" id="nav-menu">
+        <li>
+            <a href="/./">Home</a>
+        </li>
+        <li>
+            <a href="/SplitBill">Split a bill</a>
+        </li>
+        <li>
+            <a href="/LookBill">Show bills</a>
+        </li>
+    </ul>
+    <div class="logo-container">
+        <a href="/./">
+            <img src="/better_logo.png" alt="Logo" class="logo"/>

```

Comment:
`alt` text should be descriptive. Just `Logo` is probably not enough. Something like `Rubees Eat logo` might be better. In this case, in the `nav` bar it might be more apparent from context be if we can be specific we should be.